server/settings/tools.py

```diff
@@ -118,7 +118,19 @@ class WorkfilesLockProfile(BaseSettingsModel):
     enabled: bool = SettingsField(True, title="Enabled")
 
 
+class AyonMenuModel(BaseSettingsModel):
+    version_up_workfile: bool = SettingsField(
+        False,
+        title="Version Up Workfile",
+        description="Add 'Version Up Workfile' to AYON menu"
+    )
+
+
 class WorkfilesToolModel(BaseSettingsModel):
+    ayon_menu: AyonMenuModel = SettingsField(
```

Comment:
This is me being stupid: Why it's called `ayon_menu` and lives in wokfiles tool settings?

If we would like to add another actions in future to `ayon_menu` that are not workfiles related we would still add it here?

Reply:
Good question - likely best to be moved to a more standalone section.

Reply:
any suggestion where it should be moved to?